deployment/helm/node-feature-discovery/values.yaml

```
@@ -126,6 +126,11 @@ master:
     value: ""
     effect: "NoSchedule"
 
+  podDisruptionBudget:
+    enabled: false
```

For consistency, I'd suggest s/enabled/enable/ as all the other similar Helm values are named `enable`, like `gc.enable`.

```suggestion
    enable: false
```

---

Fixed.